Studies/DNN/configs/config.toml

```diff
@@ -0,0 +1,61 @@
+
+[meta]
+results_dir = "/afs/cern.ch/user/a/ayeagle/H_mumu/Studies/DNN/results"
+use_cuda = true
+
+[dataloader]
+# Events selection
+selection_cut = '(VBF_JetVeto) & (Signal_Fit)'
+# Event splitting (training_size is what's left)
+valid_size = 0.20
+test_size = 0.0
+# Input vector def
+columns_config = '/afs/cern.ch/user/a/ayeagle/H_mumu/Studies/DNN/ds_setup/general.yaml'
+signal_types = ['VBFH', 'ggH']
```

Comment:
This should be also stored in the other config ('global.yaml') 
In general: if there are some repetitions between configs try to merge them and take info from the global params (once you will define Setup --> see prev comment)